src/utils/keyManagement.js

```diff
@@ -24,24 +24,20 @@ export const eraseHpAdminKeypair = () => {
 // Create keypair using wasm-based HpAdminKeypair Class
 // Use singleton pattern
 // Return null when trying to initialize with no params
-export const getHpAdminKeypair = async (email = undefined, password = undefined) => {
+// TODO: the default email and password here are for testing only and should be removed before merging
+export const getHpAdminKeypair = async (email = "test@test.com", password = "passw0rd") => {
```

Comment:
Are you still going to change this before merging?

Reply:
good catch 👍